docs/user-manuals/cloneset.md

```diff
@@ -270,6 +273,10 @@ When `partition` is set during update:
 - If it is a number: `(replicas - partition)` number of pods will be updated with the new version.
 - If it is a percent: `(replicas * (100% - partition))` number of pods will be updated with the new version.
 
+**FEATURE STATE:** Kruise v1.2.0
+- If `partition` is a percent, and `partition < 100% && replicas > 1` , CloneSet will ensure **at least one pod** will be updated with the new version.
+- For convenience, just read `.status.expectedUpdatedReplicas` field if you don't know how to calculate the number of the pods that will be updated.
```

Comment:
> For convenience, just read `.status.expectedUpdatedReplicas` field if you don't know how to calculate the number of the pods that will be updated.

One can compare .status.expectedUpdatedReplicas and .status.expectedUpdatedReplicas to decide whether workload had finish rolling out new revision under partition restriction.

Reply:
done